lighthouse-core/test/results/sample_v2.json

```diff
@@ -2026,6 +2026,16 @@
               "selector": "h2",
               "path": "3,HTML,1,BODY,5,DIV,0,H2",
               "snippet": "<h2>Do better web tester page</h2>",
+              "boundingRect": {
+                "x": 8,
+                "y": 335,
```

Comment:
ah, these are only here because a year ago Matt hardcoded these properties in `artifacts.json` (or perhaps ran the gatherer with code that was setting `x`, `y`).

I'll remove.